README.md

```diff
@@ -191,7 +191,9 @@ All keys written by Funnel Rocket are prefixed by 'frocket:'. To configure this
 Funnel Rocket can also be configured to use a non-default logical DB for further namespace separation (meaning, db number > 0) by setting `export FROCKET_REDIS_DB=<logical db number>`. 
 
 #### Running the Worker and API Server
-
+<!-- This fails locally since the dockerised port is 6380, do we maybe want to add an option to launch it via the `docker-compose`/revert to running with the default settings/add an option to set the redis port in this tutorial ?  
+This for example works: `FROCKET_REDIS_PORT=6380 python3 -m frocket.worker.impl.queue_worker`
+-->
 To run a worker waiting on the Redis queue for tasks: `python -m frocket.worker.impl.queue_worker`.
```

Comment:
<!-- This fails locally since the dockerised port is 6380, do we maybe want to add an option to launch it via the `docker-compose`/revert to running with the default settings/add an option to set the redis port in this tutorial ?  
This for example works: `FROCKET_REDIS_PORT=6380 python3 -m frocket.worker.impl.queue_worker`
-->